ReportPortal.Tests/ReportPortal.E2E.API.Business/StepDefinitions/ApiControllers/ProjectController.cs

```diff
@@ -1,12 +1,11 @@
-﻿using System.Net.Http.Json;
-using Microsoft.Extensions.Logging;
+﻿using Microsoft.Extensions.Logging;
 using ReportPortal.E2E.API.Business.Models;
 
 namespace ReportPortal.E2E.API.Business.StepDefinitions.ApiControllers
 {
     public partial class ReportPortalApiSteps
     {
-        public Task<HttpResponseMessage> CreateProject(string projectName, string entryType = "INTERNAL")
+        public T CreateProject<T>(string projectName, string entryType = "INTERNAL")
```

Comment:
May be better to use for entryType Enum?

Reply:
Added enum